src/_pytest/compat.py

```diff
@@ -176,8 +176,9 @@ def getfuncargnames(
         p.name
         for p in parameters.values()
         if (
-            p.kind is Parameter.POSITIONAL_OR_KEYWORD
-            or p.kind is Parameter.KEYWORD_ONLY
+            # TODO: Remove type ignore after https://github.com/python/typeshed/pull/4383.
```

Comment:
The `.` at end of line breaks the link, FYI.

And perhaps we could add `warn_unused_ignores = True` to the config so that this will be caught by CI when we upgrade?

Reply:
> FYI the . at end of line breaks the link for me.

Removed.

>  And perhaps we could add warn_unused_ignores = True to the config so that this will be caught by CI when we upgrade?

We can't use `warn_unused_ignores` because there are some warnings that only happen on some Python versions, or some platforms, etc. which will then trigger the warning on others.

What I've done up to now is that when I upgrade the mypy version we use, I set it temporarily and fix what it brings up.